### PR TITLE
🦄 FIX - Helm perms 🔥

### LINKED ISSUE #365: TOOL-BOX: Helm permission issue
When I try to use Helm in the tool-box image I get the following
```
sh-4.4$ helm version
panic: /home/tool-box/.cache/helm/registry/config.json: stat /home/tool-box/.cache/helm/registry/config.json: permission denied

goroutine 1 [running]:
main.newRootCmd(0xc000232500, 0x1be8b60, 0xc0000be008, 0xc0000b0030, 0x1, 0x1, 0x0)
        /home/circleci/helm.sh/helm/cmd/helm/root.go:196 +0xfbe
main.main()
        /home/circleci/helm.sh/helm/cmd/helm/helm.go:68 +0xb4
```

### PR DESCRIPTION
**Pull request by springdo**
#### What is this PR About?
Fixes #365

#### How do we test this?
build it, run it, execute helm :)
image

cc: @redhat-cop/day-in-the-life

### CODE REVIEW

**Comment by springdo** [comment]
/assign @pabrahamsson 

As per our Chat on Google, I moved the chown down to the RUN below

File: tool-box/Dockerfile [hunk 1]
```diff
@@ -37,7 +37,8 @@ WORKDIR $HOME
 
 ADD ./root /
 
-RUN chmod u+x /usr/local/bin/run
+RUN chmod u+x /usr/local/bin/run && \
+    chown -R 1001:1001 $HOME
```
Comment (pabrahamsson) [inline tool-box/Dockerfile:41]: It appears this is still an issue when running on OCP.
```
bash-4.4$ helm version
panic: /home/tool-box/.cache/helm/registry/config.json: stat /home/tool-box/.cache/helm/registry/config.json: permission denied

goroutine 1 [running]:
main.newRootCmd(0xc0004af000, 0x1d28ec0, 0xc00000e018, 0xc00000c0b0, 0x1, 0x1, 0x0)
        /home/circleci/helm.sh/helm/cmd/helm/root.go:179 +0xc16
main.main()
        /home/circleci/helm.sh/helm/cmd/helm/helm.go:73 +0xb7
bash-4.4$ ls -al
total 0
drwxrwxr-x. 1 1001 1001 20 Jun  5 13:59 .
drwxr-xr-x. 1 root root 22 Jun  5 13:58 ..
drwx------. 1 1001 1001 17 Jun  5 13:59 .cache
bash-4.4$ id
uid=1003230000(tool-box) gid=0(root) groups=0(root),1003230000
```
I've successfully tested with replacing the `chown` line with `rm -rf $HOME/.cache` and I now get the expected output. Oh and I updated Helm to `3.2.2` (released yesterday) :boom: 
```
bash-4.4$ helm version
version.BuildInfo{Version:"v3.2.2", GitCommit:"a6ea66349ae3015618da4f547677a14b9ecc09b3", GitTreeState:"clean", GoVersion:"go1.13.12"}
```
Are you able to verify @springdo ?

**Comment by springdo** [comment]
@pabrahamsson - I've bumped the version again and applied your fix. Works locally and i ran a build and run in s44 and seems like it's working alright 
image

**Pull request merged**